notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    demo = gr.Interface(
</pre><p>please disable flagging for interface</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Flagging disabled</p>